website/src/models/group.js

```diff
@@ -8,6 +8,8 @@ var logging = require('../libs/logging');
 var Challenge = require('./../models/challenge').model;
 var firebase = require('../libs/firebase');
 
+const PARTY_CAP = 30;
```

Comment:
Actually, there's no need to export it since it's a virtual property on the group.